docs/logs/faq.md

```diff
@@ -606,28 +606,23 @@ later.
 
 ### What are Logs App Alerts?
 
-Logs App alerts are based on saved searches that trigger alert
-notifications when these saved searches meet a certain threshold
+Logs App alerts are based on full text search queries and filters. They trigger alert
+notifications when these query and filters meet a certain threshold
 condition in your logs - e.g. less than N matches in 5 minutes, more
-than M matches in 3 minutes, etc., or when such saved searches detect
+than M matches in 3 minutes, etc., or when search query detect
```

Comment:
than M matches in 3 minutes, etc., or when search query detect
=>
than M matches in 3 minutes, etc., or when search the query detects

Reply:
I don't think we meant "search the query" it is "search query". Of course, detects vs detect makes sense